fulltext/backends/__eml.py

```diff
@@ -3,10 +3,11 @@
 import codecs
 
 from six import StringIO
+from fulltext import BaseBackend
 
 
-def handle_fobj(f, **kwargs):
-    encoding, errors = kwargs['encoding'], kwargs['encoding_errors']
+# This is used by other modules, hence it's here.
+def handle_fobj(f, encoding, errors):
     text = StringIO()
     f2 = codecs.getreader(encoding)(f, errors=errors)
```

Comment:
Can you rename `f2` to `enc_f` or similar?

Also, can you flip this so that:

```python
def handle_fobj(f, enc, enc_err):
    return Backend(enc, enc_err).handle_fobj(f)
```

Reply:
I don't think it's a good idea to reinstantiate a new Backend from in here. That logic lives in `__init__.py` which currently passes 3 args, not 2. In future (e.g. with title feature) it's likely that we want to pass even more. 

> Can you rename f2 to enc_f or similar?

sure